ocfweb/static/js/pages/account/vhost_mail.js

```diff
@@ -175,5 +175,9 @@
                 showAddAddressModal($(this).data('domain'), true);
             }.bind(this));
         });
+
```

Comment:
Wow, a whole newline, that's a hard line of JS to write...

Reply:
OK, maybe it was only 3 lines of JS.